cmake/projects/cppast/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    cppast
+    VERSION
+    0.0.1-dbbbe62
+    URL
+    "https://github.com/cpp-pm/cppast/archive/dbbbe626e4cff9da83b1c0baba650fdd56df0cdc.zip"
```

Comment:
Please change the archive URL to https://github.com/cpp-pm/cppast/archive/v0.0.0-b155d6a-p0.tar.gz and the version to `0.0.0-b155d6a-p0`. The version is based on the upstream hash, not the hash of the hunterization commit.

Reply:
Ok, thanks for the hint. Done.